### DISCUSSION TITLE
Problema para ver el contenido de GITHUB.

### DISCUSSION THREAD

**Discussion by CristopherSandoval** [Launch-X-Latam/MisionFrontEnd#594] [Team Posts]
Hola compañeros! Tengo un problema al ver los temas en este repo:
https://github.com/Launch-X-Latam/MisionBackend

me aparece: error 404 repo not found.

Alguien sabe por qué? 

 también ya dejaron de aparecer los en vivo grabados :(

**Comment by Carlopsd** [comment]
Lo del GitHub es porque no se a actualizado el repositorio por el mision comander, en el archivo sesta puesto un link hacia la información pero como no se ha subido por lo que no existe pone un error 404 de que no encuentra la página.

**Comment by CristopherSandoval** [comment]
entonces no hay manera de verlo/el mission commander nunca lo actualizó? :(

**Comment by Carlopsd** [comment]
No, en su repositorio dice que la ultima actualización fue hace un mes https://github.com/josejesusguzman/MisionBackend 

Y a mi si me aparecen las grabaciones, si no las vez solo vuelve a entrar  al link del día de la clase

![grabaciones](https://user-images.githubusercontent.com/111400282/210692500-334763a4-ff78-44d9-8e7a-1d8bb679670e.png)

**Comment by CristopherSandoval** [comment]
YAAAA es que no había terminado de ver el otro live ni tampoco leí bien XD

**Comment by Drako9159** [comment]
Lo del repositorio pues la mayoría de los recursos se quedaron pendientes, supongo que no se actualizarán.

**Comment by doguedogue** [comment]
los live ahi están es cuestion que primero entres con tu cuenta de innovaccion a office.com, si tienes de outlook, live u otra cuenta de microsoft luego como que se hace bolas en el login por el "single sign on", lo ideal es hacerlo en otro navegador o modo incognito si es tu caso, asi encontrarás los videos.
